revista-ladinia.csl

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0">
+  <info>
+    <title>Revista Ladinia</title>
+    <title-short>LADINIA</title-short>
+    <id>http://www.zotero.org/styles/revista-ladinia</id>
+    <link href="http://www.zotero.org/styles/revista-ladinia" rel="self"/>
+    <link href="http://www.zotero.org/styles/apa" rel="template"/>
+    <link href="https://micura.it/la/ativites/ladinia/style-sheet" rel="documentation"/>
+    <author>
+      <name>Samuel Frontull</name>
+      <email>samuel.frontull@uibk.ac.at</email>
+    </author>
+    <author>
+      <name>Paolo Anvidalfarei</name>
+      <email>paolo@micura.it</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="linguistics"/>
+    <issn>1124-1004</issn>
+    <updated>2022-04-22T12:16:36+01:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="it">
+    <terms>
+      <term name="and others"/>
+      <term name="no date">s.a.</term>
+      <term name="no-place">s.l.</term>
+      <term name="no-publisher">N.N.</term>
+      <term name="editor">
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <locale xml:lang="de">
+    <terms>
+      <term name="and others"/>
+      <term name="no date">s.a.</term>
+      <term name="no-place">s.l.</term>
+      <term name="no-publisher">N.N.</term>
+      <term name="editor">
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <locale xml:lang="en">
+    <terms>
+      <term name="and others"/>
+      <term name="no date">s.a.</term>
+      <term name="no-place">s.l.</term>
+      <term name="no-publisher">N.N.</term>
+      <term name="editor">
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <choose>
+      <if variable="author">
+        <names variable="author">
+          <name form="short" et-al-use-first="1">
+            <name-part name="family" font-variant="small-caps"/>
+          </name>
+          <et-al term="and others"/>
+        </names>
+      </if>
+      <else-if variable="editor">
+        <names variable="editor">
+          <name name-as-sort-order="all" delimiter="/">
+            <name-part name="family" font-variant="small-caps"/>
+          </name>
+          <label prefix=" (" suffix=")"/>
```

Comment:
I did not use substitute because it did not work as expected in the case where no authors and no editors are specified.
For that, I would need a nested `substitute`, right? It does not work if I define it as follows:

```
<names variable="author">
  <name name-as-sort-order="all" form="short">
    <name-part name="family" font-variant="small-caps"/>
  </name>
  <et-al term="and others"/>
  <substitute>
    <names variable="editor">
      <name name-as-sort-order="all" delimiter="/">
        <name-part name="family" font-variant="small-caps"/>
      </name>
      <et-al term="and others"/>
      <substitute>
        <text term="anonymous"/>
      </substitute>
    </names>
  </substitute>
</names>
```

Reply:
No, you can have multiple entries in substitute. The first matching one is used. 

You can just use
```
<substitute>
  <names variable="editor"/>
  <text term="anonymous"/>
</substitute>
```

The options on `<names>` are inherited from the parent `<names>` so you don't have to specify them again.

Reply:
![Screenshot from 2022-04-25 13-51-50](https://user-images.githubusercontent.com/10232781/165083748-437c2013-b840-4956-bd8c-67603e9ff97d.png)

Here it should display `(N.N. 1980)` and `N.N.: Book without author, s.l. 1980.` but it doesn't...

Reply:
I guess this issue only occurs inside the Zotero Style Editor, I just saw that the generated sample translations below are as expected.

Reply:
No, this is a bit of a complex issue related to groups. You'll need to put the anonymous term in a macro and call that in substitute for the style to work properly in Zotero, Mendeley, and other tools using the same citation processor (the one we use for previews here is slightly different).